staff/sergiomons/app-template/styles/landing.css

```diff
@@ -0,0 +1,90 @@
+@import "common1.css";
+.content {
+    display: grid;
+    height: 100%;
+    /*grid-template-columns: repeat(auto-fit, minmax(40%, 1fr));*/
+    grid-template-columns: 37% 63%;
+    grid-template-rows: 7% 25% 17% 6% 15% 20%;
+    color:white;
+    font-weight: bold;
+    text-align: center;
+}
+.content > .p-intro {
+    grid-row: 3;
+    font-size: 190%;
+}
+.content> .btns {
+    display: flex;
+    justify-content: center;
+    grid-row: 4;
+}
+.content > .p-regist {
+    grid-row: 5;
+    font-size: 130%;
+}
+
+/*.buttons {
```

Comment:
!!!!!! 

<img src="https://i.giphy.com/media/5xaOcLwEvFOizxHVyVy/giphy.webp" />

Reply:
Jejeje yes, i forget delete them. I have done it!